docs/src/apps/jsonrpc-api.md

```diff
@@ -403,6 +404,35 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 {"jsonrpc":"2.0","result":{"firstNormalEpoch":8,"firstNormalSlot":8160,"leaderScheduleSlotOffset":8192,"slotsPerEpoch":8192,"warmup":true},"id":1}
 ```
 
+### getFeeRateGovernor
+
+Returns the fee rate governor information from the root bank
+
+#### Parameters:
+
+None
+
+#### Results:
+
+The `result` field will be an `object` with the following fields:
+
+* `burnPercent: <u8>`, Percentage of fees collected to be destroyed
+* `lamportsPerSignature: <u64>`, Lamports charged as a fee for each signature on a transaction
```

Comment:
Ideally `lamportsPerSignature` shouldn't be returned here, since we have no corresponding blockhash

Reply:
That's a `#[serde(skip)]`, right?

Reply:
Yep.  That, until we have a conflicting serialization requirement at which point we'd add a RpcFeeRateGovernor to rpc_response.rs